.drone.yml

```diff
@@ -66,7 +68,9 @@ steps:
     ignore_missing: true
   when:
     event:
-    - tag
+      - tag
+    instance:
+      - drone-publish.rancher.io
 depends_on:
 - linux-amd64
 ...
```

Comment:
Sorry to be a nick-picker, could you add a new line at the bottom of the file?

Reply:
Done!